src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.html

```diff
@@ -1,8 +1,13 @@
 <ul class="edit-page-nav">
-    <li *ngFor="let item of model | async">
-        <a class="edit-page-nav__item" [class.edit-page-nav__item--disabled]="item.disabled" routerLinkActive="edit-page-nav__item--active" [routerLink]="item.link" [queryParams]="route.queryParams | async">
+    <li *ngFor="let item of model | async" >
+        <a *ngIf="!item.needslicense" class="edit-page-nav__item" [class.edit-page-nav__item--disabled]="item.disabled" routerLinkActive="{{ !item.disabled ? 'edit-page-nav__item--active' : '' }}" [routerLink]="!item.disabled ? item.link : null" [queryParams]="route.queryParams | async">
```

Comment:
You can do if else in templates now: https://stackoverflow.com/questions/43006550/how-to-use-ngif-else-in-angular?rq=1